modules/using-aws-cli-create-iam-role-alb-controller.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * networking/installing-albo-sts-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="using-aws-cli-create-iam-role-alb-controller_{context}"]
+= Using the AWS CLI to create an IAM role for the Controller
+
```

Comment:
Procedure modules must have a brief introduction: https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs. Also if you mention the controller in the introductory text, I think you should use the full controller name `AWSLoadBalancerController`

Reply:
D'oh, can't believe I consistently forgot about this, haha. :sweat_smile: 

Thanks for the reminder!